langkit/templates/python_api/playground_py.mako

```diff
@@ -26,75 +25,31 @@ there are multiple.
 Enjoy!
 """.strip()
 
+class Playground(${module_name}.App):
 
-
-class Playground(object):
-    """
-    Base class to regroup logic for the playground. We use a class so that
-    specific languages implementations can add specific arguments and
-    processing by overriding:
-
-    - add_arguments to add arguments to the argparse.Parser instance
-
-    - create_unit_provider to return a custom unit provider to be used by the
-      AnalysisContext.
-    """
-
-    def __init__(self):
-        self.parser = argparse.ArgumentParser(
-            description="${module_name} playground. Analyze files passed "
-            "as arguments."
-        )
-        self.parser.add_argument('files', nargs='+', help='Files')
+    def add_arguments(self):
         self.parser.add_argument(
             '-i', '--input-script', type=str, default='',
             help="Script to execute when playground has loaded the units"
         )
-
-    def add_arguments(self):
-        """
-        Hook for subclasses to add arguments to self.parser. Default
-        implementation does nothing.
-        """
-        pass
-
-    def create_unit_provider(self):
-        """
-        Hook for subclasses to return a custom unit provider.
-        Default implementation returns None.
-        """
-        return None
-
-    def process_file(self, file_name):
-        u = self.ctx.get_from_file(file_name)
-        return u
+        super(Playground, self).add_arguments()
 
     def main(self):
-        self.add_arguments()
-        self.args = self.parser.parse_args()
-        self.ctx = ${module_name}.AnalysisContext(
-            'utf-8', with_trivia=True,
-            unit_provider=self.create_unit_provider()
-        )
+        self.process_files()
+        print HEADER
```

Comment:
This makes me realize that this file is missing our `from __future__` imports. Could you add them? (so that Python itself detects the lack of parens above)